src/polygonFeature.js

```diff
@@ -392,6 +401,89 @@ var polygonFeature = function (arg) {
     return result;
   };
 
+  /**
+   * Take a set of data, reduce the number of vertices per polygon using the
+   * Ramer–Douglas–Peucker algorithm, and use the result as the new data.
+   * This changes the instance's data, the position accessor, and the polygon
+   * accessor.
+   *
+   * @param {array} data A new data array.
+   * @param {number} [tolerance] The maximum variation allowed in map.gcs
+   *    units.  A value of zero will only remove perfectly colinear points.  If
```

Comment:
I am wondering if the unit should be pixels for the tolerance?

Reply:
It defaults to 0.5 pixel at the current zoom.  But if you are overriding this, you might not want the tolerance to be dependent on the current scale of the map.  So this can be made into current zoom pixels via `map.unitsPerPixel(map.zoom()) * <tolerance in pixels>`, or it can be in gcs units.

Reply:
so the user has to pass t = map.unitsPerPixel(map.zoom()) * <tolerance in pixels> or else we will use map.unitsPerPixel(map.zoom()) * 0.5?

Reply:
I see. I am wondering if we could auto compute it given some initial parameter in a separate PR (as a new feature), not needed now, but wanted to mention it.